### PR TITLE
Add id (Bahasa Indonesia) translation

### PR DESCRIPTION
**Pull request by ardyfeb**
Added Indonesian translation

### CODE REVIEW

File: src/locales/id.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    "lock": "Biarkan alat yang dipilih aktif setelah menggambar"
+  },
+  "headings": {
+    "canvasActions": "Aksi Kanvas",
+    "selectedShapeActions": "Selected shape actions | Aksi bentuk yang dipilih",
+    "shapes": "Bentuk"
+  },
+  "hints": {
+    "linearElement": "Klik untuk memulai banyak poin, seret untuk satu baris",
+    "linearElementMulti": "Press Escape or Enter to finish | Tekan Escape atau Enter untuk menyelesaikan",
```
Comment (j-f1) [inline src/locales/id.json]: The “Press Escape or enter to finish” part can be removed, correct?

File: src/locales/id.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    "canvasBackground": "Latar Kanvas",
+    "drawingCanvas": "Drawing Canvas",
+    "layers": "Lapisan",
+    "language": "Bahasa"
+  },
+  "buttons": {
+    "clearReset": "Setel Ulang Kanvas",
+    "export": "Ekspor",
+    "exportToPng": "Expor ke PNG",
+    "exportToSvg": "Expor ke SVG",
```
Comment (j-f1) [inline src/locales/id.json]: I don’t speak Indonesian but I wanted to make sure this wasn’t a typo (“Ekspor” above, “Expor” here).

File: src/locales/id.json (hunk 1 above)
Comment (ardyfeb) [reply src/locales/id.json]: Oh i see, that is i used for placeholder but i forgot to remove that

File: src/locales/id.json (hunk 2 above)
Comment (ardyfeb) [reply src/locales/id.json]: Fixed in next commit

**Comment by lipis** [comment]
@ardyfeb Could you join this project and fix the translations there https://crowdin.com/project/excalidraw/id?

We are using Crowdin for translations

**Comment by ardyfeb** [comment]
> @ardyfeb Could you join this project and fix the translations there https://crowdin.com/project/excalidraw/id?
> 
> We are using Crowdin for translations

Got it !

**Comment by lipis** [comment]
Closing this in favor of #845

**Pull request closed without merging**